examples/distinct-object-types/distinct_object_types.bal

```diff
@@ -1,30 +1,53 @@
 import ballerina/io;
 
-// The `Person` object type that contains a string field called `name`.
-type Person distinct object {
+// DistinctPerson is a proper subtype of Person
+class Person {
     public string name;
+
+    function init(string name) {
+        self.name = name;
+    }
 };
 
-// The `Engineer` and `Manager` classes are structurally the same but introducing the
-// `distinct` keyword distinguishes them by considering them as nominal types.
-distinct class Engineer {
-    *Person;
+distinct class DistinctPerson {
+    public string name;
 
     function init(string name) {
         self.name = name;
     }
 }
 
-distinct class Manager {
-    *Person;
+// SomeWhatDistinctPerson is a subtype of DistinctPerson since inherit the same type ID via inclusion
```

Comment:
```suggestion
// The `SomeWhatDistinctPerson` type is a subtype of the `DistinctPerson` type
// since it includes the `DistinctPerson` type's type IDs via inclusion.
```